apps/allocations/app/components/App/AllocationsHistory.js

```diff
@@ -12,9 +13,18 @@ import PropTypes from 'prop-types'
 
 import { STATUSES } from '../../utils/constants'
 import { displayCurrency } from '../../utils/helpers'
+import { match } from 'minimatch'
 
 const AllocationsHistory = ({ allocations }) => {
   const theme = useTheme()
+  const { balances = [] } = useAppState()
+  const getTokenSymbol = inputAddress => balances.reduce((matchingSymbol,
+    { symbol, address }) => {
+    if (inputAddress === address) return symbol 
+    else return matchingSymbol
```

Comment:
ahhh I'm breaking the policy of "re: style comments, add a linting rule or gtfo", but breaking up those args onto multiple lines makes my eyes cross. How about this:

```js
const getTokenSymbol = inputAddress => balances.reduce(
  (matchingSymbol, { symbol, address }) => inputAddress === address ? symbol : matchingSymbol,
  ''
)
```

Wait, now that I can see it better, isn't it more straightforward to use `find`?

```js
const getTokenSymbol = inputAddress => {
  const matchingBalance = balances.find(({ address }) => inputAddress === address)
  return matchingBalance ? matchingBalance.symbol : ''
}